src/students/students.controller.ts

```diff
@@ -76,4 +83,20 @@ export class StudentsController {
       user,
     );
   }
+
+  @ApiOperation({ summary: 'Return an array of all students' })
+  @ApiOkResponse(getAllStudentsOkResponse)
+  @HttpCode(HttpStatus.OK)
+  @Get('/all-students')
```

Comment:
jesteśmy już w kontrolerze `students` nie musimy nazywać ścieżki `all-students`, wystarczy `all`, albo w ogóle `/` - konwencja tworzenia REST API mówi:
- tworzenie `/students` POST
- pobieranie wszystkich `/students` GET
- pobieranie pojedynczego `/students/:id` GET
-  i tak dalej :)

a że już mamy kontroller `students` to zostaje tylko dodać same `/`